05-jupyter-ecosystem.md

```diff
@@ -371,7 +371,7 @@ with the `%matplotlib notebook` or `%matplotlib inline` cell
 magics. 
 * Pandas
 ([https://pandas.pydata.org/](https://pandas.pydata.org/)) provides
-resources for data analysis and a flexible data structures for labeled two-dimenstional data. 
+resources for data analysis and a flexible data structures for labeled tabular data. 
```

Comment:
While you're on this line, "and a flexible data structure" or "and flexible data structures"; optionally mention IO, time series, and/or other notable functionality.

Reply:
Opened #148 to cover @jedbrown's suggestion. Going to merge this PR.